docs/features/cct.md

```diff
@@ -21,7 +21,7 @@ Accurate | This mode subtracts the calculated white value from the RGB channels.
 Brighter | This does the exact opposite and not touch the RGB channels at all, just mix in the dedicated white.
 None | No auto white calculation is performed. The white channel of colors can be manually set using the `White channel` slider in the user interface, RGB-only effects and most realtime sources will leave the white channel off.
 Dual | The `White channel` slider is present in the UI and works the same as in `None` mode, however if the slider value is 0 (far left), the `Brighter` mode is used for auto white calculation. This is the default auto white mode.
-Max | Sets white to the value of the brightest RGB channel. This is good for white-only LEDs without any RGB.
+Max | Sets white to the value of the brightest RGB channel. This is good for white-only LEDs without any RGB, however note that using any value other than "None" will cause the light to be treated as having RGB channels, so the color picker will show in the web UI.
```

Comment:
This is true for any mode, not just Max. Perhaps move it out of the spreadsheet.

Reply:
> This is true for any mode, not just Max. Perhaps move it out of the spreadsheet.

I mentioned this here because "Max" is the only option where it specifically says that it is "good for white-only LEDs". Or are you saying that the part about it being good for white-only LEDs should be removed?

Reply:
You seem obsessed with "white only". 😄 Sorry, no offense intended.
Any option is good (or possible) for "white only", depending on what you need or want (or what your external devices expect, yes don't forget that).
I'm saying that choosing anything but "None" will treat your output as RGB and not RGBW or W.

Reply:
> You seem obsessed with "white only". 😄 Sorry, no offense intended.

i'm just trying to understand. As a new user, it is confusing to me to see a full RGB color picker when I only have plain white LEDs connected, have set all the outputs to be white, and chosen the white handling option that is documented to be good for white-only LEDs. I can't imagine I'm the only person in the whole world who is confused by this, so I'm trying to figure it out and hopefully update the documentation so it's more clear to other new users.

Reply:
What further adds to the confusion is that if I set the first 3 outputs to "None" and the 4th output to "Max", I don't get the color wheel, but if I set any of the other outputs to "Max" then I do get the color wheel. That seems like an actual bug to me. This is with the [build](https://github.com/blazoncek/WLED/actions/runs/14008917083) you asked me to try the other day though, so maybe it's related to that. I can try one of the stable builds if you want.

Reply:
The UI adapts to the **main** segment. Any segment can be *main* and it is distinguished by a lighter gray in UI. Also affecting UI are selected segments (the ones with checkmark).

Reply:
I only have one segment, which contains all 4 outputs.

Reply:
Then you have your answer. If one of the outputs is RGB (as when Auto white is enabled) UI will respond as segment being RGBW, separate outputs into segments and you'll have control individually.
You can automatically create segments from outputs using selection box in settings.

Please continue discussion either on forum or on Discord.